src/pykiso/test_setup/dynamic_loader.py

```diff
@@ -229,6 +229,12 @@ def get_instance(self, name: str) -> Union[AuxiliaryCommon, Connector]:
         log.internal_debug(f"instantiated {name}")
         return inst
 
+    def delete_all_instances(self) -> None:
+        """Call custom del method if it exists"""
+        for instance in self.instances.values():
```

Comment:
I still feel like this could be achieved more elegantly, in `del` doesn't call `__del__` it means that you still have references to the cchannels, it would be good to know where